modules/analytics/src/views/full/index.tsx

```diff
@@ -201,6 +217,37 @@ const Analytics: FC<any> = ({ bp }) => {
     return data.metrics
   }
 
+  const fetchQnaQuestions = async () => {
+    const qnaIds = orderMetrics(getMetric('msg_sent_qna_count'))
+      .filter(m => m.name)
+      .slice(0, 10)
+      .map(m => m.name)
+
+    const fetchedQuestions = await Promise.all(
+      qnaIds.filter(id => !(id in qnaQuestionsCache)).map(id => fetchQnaQuestion(id.replace('__qna__', '')))
+    )
+
+    for (const {
+      data: { questions },
+      id
+    } of fetchedQuestions) {
+      const question = questions[lang.getLocale()][0]
```

Comment:
The UI has 2 languages (EN/FR) while a bot could be in a lot of different languages (ES, AR, for example). Therefore, you should check if questions has the getLocale key, and if not, use the default language (or any language available)

Reply:
Fixed in d4ea723